samples/attested_tls/README.md

```diff
@@ -71,7 +71,55 @@ Note: Both of them can run on the same machine or separate machines.
 
 ## Build and run
 
-To build and run the samples, refer to documentation in the main [README file](../README.md#building-the-samples.md).
+### Linux
+
+#### CMake
+- Use Mbed TLS
+  ```bash
+  mkdir build
+  cd build
+  cmake -DOE_CRYPTO_LIB=mbedtls ..
+  make run
+  ```
+- Use OpenSSL
+  ```bash
+  mkdir build
+  cd build
+  cmake -DOE_CRYPTO_LIB=openssl ..
+  make run
+  ```
+
+#### GNU Make
+- Use Mbed TLS
+  ```bash
+  make OE_CRYPTO_LIB=mbedtls build
```

Comment:
Will this override internal OE_CRYPTO_LIB setting?

Reply:
Yes. But the behavior is the same as the default given that OE_CRYPTO_LIB is set to the same value.